lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts

```diff
@@ -52,4 +53,64 @@ export class LegalHoldApi extends BaseApi {
             returnType: HoldPaging
         });
     }
+
+    /**
+     * Assign hold of legal holds
+     *
+     * @param holdId The identifier of a hold.
+     * @param ids one element list with id of hold to assign to existing hold
+     * @returns Promise<HoldEntry>
+     */
+    assignHold(ids: [string], holdId: string): Promise<HoldEntry> {
+        throwIfNotDefined(holdId, 'holdId');
+        throwIfNotDefined(ids, 'ids');
+
+        return this.post({
+            path: `/holds/{holdId}/children`,
+            pathParams: { holdId },
+            bodyParam: ids.map((id) => ({
+                id
+            })),
```

Comment:
could you please provide ids with interface that needed to this endpoint?
` assignHold(ids: {id: string}[], holdId: string)`

the same for bulk action